Dockerfile

```diff
@@ -70,7 +70,7 @@ RUN if ! getent group "$USER_GID"; then groupadd --gid "$USER_GID" dependabot ;
 ### RUBY
 
 # When bumping Ruby minor, need to also add the previous version to `bundler/helpers/v{1,2}/monkey_patches/definition_ruby_version_patch.rb`
-ARG RUBY_VERSION=3.1.3
+ARG RUBY_VERSION=3.2.0
```

Comment:
We restructured the dockerfiles, so this now needs to be bumped over here instead:
https://github.com/dependabot/dependabot-core/blob/3313530fa05c482dc0dace53af7dfffee185e2fd/Dockerfile.updater-core#L40

Can you rebase to pickup that change?

Reply:
@jeffwidman done.